agent/exec/container/container.go

```diff
@@ -12,6 +13,7 @@ import (
 	"github.com/docker/docker/api/types/events"
 	"github.com/docker/docker/api/types/filters"
 	"github.com/docker/docker/api/types/network"
+	"github.com/docker/go-connections/nat"
```

Comment:
Yeah, this is just a utility package used historically to parse or construct portSpecs passed to engine-api types.